### PR TITLE
Fabo/dynamic images

### PR DESCRIPTION
**Pull request by faboweb**
images are now dynamically locally loaded again

### CODE REVIEW

File: components/portfolio/BalanceRow.vue [hunk 1]
```diff
@@ -108,7 +111,8 @@ export default {
     },
     image() {
       const coinLookup = network.getCoinLookup(this.balance.denom, 'viewDenom')
-      return coinLookup ? coinLookup.icon : undefined
+      const icon = coinLookup ? coinLookup.icon : undefined
+      return icon
```
Comment (Bitcoinera) [inline components/portfolio/BalanceRow.vue:115]: what is the difference? It looks the same

Comment (faboweb) [inline components/portfolio/BalanceRow.vue:115]:
```suggestion
      return coinLookup ? coinLookup.icon : undefined
```

**Comment by faboweb** [comment]
solution was moving files into the static folder

**Comment by Bitcoinera** [comment]
why is the branch called `fabo/ibc`? Recycling branches? :recycle:

**Comment by faboweb** [comment]
Was just continuing from when I implemented the icon logic

**Pull request merged**